test.sh

```diff
@@ -4,18 +4,28 @@ PWD=`pwd`
 
 cd test_project
 python manage.py test
+EXIT1=$?
 
+EXIT2=0
 if [ "$PY27_DJANGO17" = "0" ]; then
     while getopts ":p" opt; do
         case $opt in
             p)
                 pip install psycopg2==2.6.2
                 cd ../test_project_postgres
                 python manage.py test
+                EXIT2=$?
            ;;
         esac
     done
 fi
 
 
 cd $PWD
+
+if test $EXIT1 -eq 0 && test $EXIT2 -eq 0
```

Comment:
Can you explain please what does this fix?

Reply:
Without this change, the exit status of `test.sh` is always 0 (success). With this change, it will have exit status 0 only if both test scripts had exit status 0, otherwise it will be 1 (fail).

Without this change, tox reports success all round even if all the tests fail.